crates/router/src/connector/stripe/transformers.rs

```diff
@@ -1513,16 +1662,21 @@ impl<F, T>
             types::PaymentsResponseData,
         >,
     ) -> Result<Self, Self::Error> {
-        let redirection_data = item
-            .response
-            .next_action
-            .as_ref()
-            .map(|next_action_response| {
-                services::RedirectForm::from((
-                    next_action_response.get_url(),
-                    services::Method::Get,
-                ))
-            });
+        let redirect_data = item.response.next_action.to_owned();
+        let redirection_data =
+            if let Some(redirection) = redirect_data {
+                match redirection {
+                    StripeNextActionResponse::DisplayBankTransferInstructions(_) => None,
+                    _ => Some(services::RedirectForm::from((
+                        redirection.get_url().ok_or(
+                            errors::ConnectorError::MissingRequiredField { field_name: "url" },
+                        )?,
```

Comment:
Cannot raise this error when handling the response. We should handle the redirection if it is provided by the connector in response, if not this should be sent as none